include/battle.h

```diff
@@ -732,6 +732,8 @@ struct BattleStruct
     bool8 transformZeroToHero[PARTY_SIZE][NUM_BATTLE_SIDES];
     u8 pledgeMove:1;
     bool8 isSkyBattle:1;
+    u8 timesGotHit[PARTY_SIZE][NUM_BATTLE_SIDES];
```

Comment:
Personally I would put the battle side first and make the party size the subarray, but I see you're just following `transformZeroToHero` which was also done this way

Reply:
Out of curiosity. Why do you think it's better? I wouldn't mind changing it though this is what all fields look like in the expansion if they need party and side.

Reply:
Yeah I agree and also changed it since it looks like there is no consistency right now anyway and `transformZeroToHero` field is changed in #3614 so it should be fine.